src/pages/projects.js

```diff
@@ -1,11 +1,19 @@
+import { PageHeader } from '../components/page-header/page-header'
+import { useState } from 'react'
 import { ProjectContent } from '../components/project-content/project-content'
+import { ProjectsDatas } from '../components/projects-datas'
 export const ProjectsPage = () => {
+    const [datas, setDatas] = useState([{ ProjectsDatas }])
```

Comment:
this one needs to change into
`const [datas, setDatas] = useState(ProjectsDatas)`

Reply:
okay, changed